tests/kani/Enum/discriminant_128bits.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+// Check niche optimization for mix of option, tuple and nonnull.
```

Comment:
This comment doesn't seem accurate (don't see a tuple, and nonnull should perhaps be nonzero).

Reply:
yes, it is outdated